plugins/lightspeed-backend/src/service/router.ts

```diff
@@ -0,0 +1,88 @@
+import { errorHandler } from '@backstage/backend-common';
+
+import express from 'express';
+import Router from 'express-promise-router';
+import { APIError } from 'openai';
+
+import { RouterOptions, QueryRequestBody } from './types';
+import { validateCompletionsRequest } from './validation';
+
+
+import { ChatOpenAI } from "@langchain/openai";
+import { HumanMessage} from "@langchain/core/messages";
+import {ChatPromptTemplate, MessagesPlaceholder} from "@langchain/core/prompts";
```

Comment:
```suggestion
import { HumanMessage } from "@langchain/core/messages";
import { ChatPromptTemplate, MessagesPlaceholder } from "@langchain/core/prompts";
```

You can use pre-commit hooks to lint your code, here is more info: https://github.com/janus-idp/backstage-plugins/blob/main/CONTRIBUTING.md#lint